### PR TITLE
Add folder structure for official HF course

### PR DESCRIPTION
**Pull request by johko**
To make the course compatible with the doc-builder of HF, we need to change the structure of the course a bit.

I made some slight changes to the main README

For now this could live alongside the old structure, to avoid any crazy merge conflicts. For that reason I added a note in the old folders README, marking it as deprecated

### CODE REVIEW

File: chapters/en/Unit 4 - Mulit-Modal Models/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This unit contains all the content that formerly was split into
```
Comment (lunarflu) [inline chapters/en/Unit 4 - Mulit-Modal Models/README.md:1]: I would adjust the name to `Multimodal Models`

File: course/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+This folder mainly exists for legacy reasons and will be removed before officially releasing the course. Please don't add any new files here.
+Your mdx files should go to the `chapters` and notebooks into the `notebooks` directory.
```
Comment (lunarflu) [inline course/README.md:2]: Your `.mdx` files

**Review by lunarflu — APPROVED** [review]
Overall LGTM! cc @merveenoyan for any additional thoughts 🤗

**Review by merveenoyan — APPROVED** [review]
Thanks a lot for tackling this!

File: README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ This is the repository for a community-led course on Computer Vision. Below, you
 1. One person from your team should create a fork of this repository
 2. Add your team members as collaborators to the fork
 3. Add a branch for your section
-4. Create Jupyter Notebooks for the chapters you want to contribute to
+4. Create `.mdx` files or Jupyter Notebooks for the chapters you want to contribute to
```
Comment (merveenoyan) [inline README.md:24]: BTW I already opened a PR to clarify this.

File: README.md [hunk 4]
```diff
@@ -76,6 +76,12 @@ For example 👇
 $$Y = X * \textbf{dequantize}(W); \text{quantize}(W)$$
 ```
 
+**Colab Option**
+You can add the option to open your `.mdx` file as a notebook in Colab by adding
```
Comment (merveenoyan) [inline README.md:80]: I think this should be a separate section (sorry I saw this after already creating a PR about it) given it's reason why we will not have a lot of notebooks.

File: chapters/en/Unit 3 - Vision Transformers/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+This unit contains all the content that formerly was split into
```
Comment (merveenoyan) [inline chapters/en/Unit 3 - Vision Transformers/README.md:1]: I think we don't need to clarify with 2.2... but just with section names, WDYT?

File: README.md (hunk 3 above)
Comment (johko) [reply README.md:24]: yup, will remove my README changes, I think your's cover everything

**Pull request merged**